diff --git a/node_modules/video-stream-ascii/dist/components/VideoAscii.js b/node_modules/video-stream-ascii/dist/components/VideoAscii.js
index 9892b9b..897268f 100644
--- a/node_modules/video-stream-ascii/dist/components/VideoAscii.js
+++ b/node_modules/video-stream-ascii/dist/components/VideoAscii.js
@@ -1,6 +1,6 @@
 import React, { useRef, useEffect, useState, createRef } from 'react';
-import { asciiChars } from '../constants/pixel-ascii';
-import { calculateAndSetFontSize, canvasImgToUrl, getAsciiFromImage, getAsciiFromImageColor, lineHeight, lineSpacing, } from '../canvas-handler/video-canvas-ascii';
+import { asciiChars } from '../constants/pixel-ascii.js';
+import { calculateAndSetFontSize, canvasImgToUrl, getAsciiFromImage, getAsciiFromImageColor, lineHeight, lineSpacing, } from '../canvas-handler/video-canvas-ascii.js';
 export var ArtTypeEnum;
 (function (ArtTypeEnum) {
     ArtTypeEnum["ASCII"] = "ASCII";
